src/main/java/controller/Guide.java

```diff
@@ -1,18 +1,30 @@
 package controller;
 
 import javafx.event.ActionEvent;
-import javafx.fxml.FXMLLoader;
-import javafx.scene.Parent;
-import javafx.scene.Scene;
-import javafx.stage.Stage;
 import util.Constants;
 import util.StageManager;
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+
 
 public class Guide {
-    public void buttonTapped(ActionEvent actionEvent) throws Exception {
-        /**
-         * Open Setup
-        */
-        StageManager.changeScene(Constants.PRIMARY_STAGE, Constants.SETUP_SCENE);
+
+    private String setupIs;
+    public void initialize() throws IOException {
+        String FileName = "memo";
```

Comment:
Put the filename into Constants

Reply:
Instead of using BufferedReader directly here, make a basic Method in Record class to read the file.